configure.ac

```diff
@@ -14,8 +17,8 @@ AC_ARG_VAR([GTEST_DIR], [directory containing Google Test])
 
 # Checks for programs.
 AC_PROG_AWK
-AC_PROG_CXX
-AC_PROG_CC
+AC_PROG_CXX([g++])
+AC_PROG_CC([gcc])
```

Comment:
Whoops, no that shouldn't have been there, that was me messing around earlier on.